docs/en/concepts-and-designs/s2i_b2i_overview.md

```diff
@@ -0,0 +1,30 @@
+# KubeSphere S2I/B2I overview
+
+## What is S2I/B2I
+
+Kubesphere S2I(Source-to-Image) / B2I(Binary-to-Image) provides easy CI/CD functionalities, help developers to publish their service to Kubernetes without writing Dockefile.
+
+## Workflow Example
+
+![](https://pek3b.qingstor.com/kubesphere-docs/png/20191223103533.png)
+
+## Principle
```

Comment:
I just see the introduction about S2i, without B2i in the following section, but the title is S2I/B2I overview.  @soulseen

Reply:
update s2i arch.